webpack.config.js

```diff
@@ -132,4 +135,7 @@ module.exports = {
     ],
   },
   plugins,
+  resolve: {
+    alias: { path: false },
+  },
```

Comment:
the above mentioned polyfill. Or rather, we're creating an empty one, as we're happy we have the core module